README.md

```diff
@@ -26,6 +26,54 @@ Prerequisites:
 Installation (make)
 ------------
 
+### Installing ls1-MarDyn using cmake
+This is the recommended way of building ls1-MarDyn.
+
+#### Configuration
+To build ls1-MarDyn using cmake first create an additional directory on the root ls1-MarDyn directory and change into that directory.
+```bash
+mkdir build
+cd build
+```
+Next, `cmake` has to be executed. In most cases, you will have to specify the compiler with which ls1-MarDyn should be built:
+```bash
+CC=clang CXX=clang++ cmake ..
+```
+Some of the most common compilers are
+| Compiler      | CC    | CXX     |
+|---------------|-------|---------|
+| GCC           | `gcc`   | `g++`     |
+| Intel Classic | `icc`   | `icpc`    |
+| Intel oneAPI  | `icx`   | `icpx`    |
+| Clang         | `clang` | `clang++` |
+
+Specifying the compiler this way is only possible at the first execution of cmake.
+If you want to change the compiler later on, either add another build directory, or first clear the existing build directory.
+
+To configure the options within ls1-MarDyn it is recommended to use `ccmake`:
+```bash
+ccmake .
```

Comment:
```suggestion
ccmake ..
```
Doesn't really make a difference for REconfiguring but if you run it the first time only the version with `..` works and I'd avoid the ambiguity.

Reply:
Good point